docs/source/API/core/initialize_finalize/initialize.rst

```diff
@@ -0,0 +1,91 @@
+``initialize``
+==============
+
+.. role::cpp(code)
+    :language: cpp
+
+Defined in header ``<Kokkos_Core.hpp>``
+
+Usage: 
+
+.. code-block:: cpp
+
+    Kokkos::initialize(argc, argv);
+    Kokkos::initialize(Kokkos::InitializationSettings()  // (since 3.7)
+                       .set_disable_warnings(true)
+                       .set_num_threads(8)
+                       .set_map_device_id_by("random"));
```

Comment:
```suggestion
                           .set_disable_warnings(true)
                           .set_num_threads(8)
                           .set_map_device_id_by("random"));
```

Reply:
Hello! I had seen this difference but I wasn't sure which version to keep. It also allows me to test the comments on my PR. I'll make the change right away. Have a nice day!